src/document/DocumentAnnotator.ts

```diff
@@ -34,6 +34,7 @@ export default class DocumentAnnotator extends BaseAnnotator {
 
         this.removeListener(Event.ANNOTATIONS_MODE_CHANGE, this.handleChangeMode);
         document.removeEventListener('selectionchange', this.debounceHandleSelectionChange);
+        this.getPages().forEach(pageEl => pageEl.removeEventListener('mouseup', this.handleSelectionChange));
```

Comment:
I believe page elements can be destroyed every time `init` is called, so we may be created orphaned event handlers.